independent-projects/qute/core/src/main/java/io/quarkus/qute/CharObjectHashMap.java

```diff
@@ -0,0 +1,758 @@
+/*
+ * Copyright 2014 The Netty Project
```

Comment:
What's the deal with the header? Do we need to keep it?

Reply:
IIRC, yes.